[2024-09-30T17:43:36.709+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-30T17:43:36.738+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_green_trip manual__2024-09-30T17:37:19.589455+00:00 [queued]>
[2024-09-30T17:43:36.751+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_green_trip manual__2024-09-30T17:37:19.589455+00:00 [queued]>
[2024-09-30T17:43:36.752+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-09-30T17:43:36.766+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_staging_layer.dbt_run_green_trip> on 2024-09-30 17:37:19.589455+00:00
[2024-09-30T17:43:36.773+0000] {standard_task_runner.py:63} INFO - Started process 266 to run task
[2024-09-30T17:43:36.776+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Taxi_trip_pipeline', 'dbt_staging_layer.dbt_run_green_trip', 'manual__2024-09-30T17:37:19.589455+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/taxi_trip.py', '--cfg-path', '/tmp/tmp0xr3avn3']
[2024-09-30T17:43:36.779+0000] {standard_task_runner.py:91} INFO - Job 48: Subtask dbt_staging_layer.dbt_run_green_trip
[2024-09-30T17:43:36.828+0000] {task_command.py:426} INFO - Running <TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_green_trip manual__2024-09-30T17:37:19.589455+00:00 [running]> on host 72cf0e8bafa8
[2024-09-30T17:43:36.919+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='Taxi_trip_pipeline' AIRFLOW_CTX_TASK_ID='dbt_staging_layer.dbt_run_green_trip' AIRFLOW_CTX_EXECUTION_DATE='2024-09-30T17:37:19.589455+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-30T17:37:19.589455+00:00'
[2024-09-30T17:43:36.921+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-30T17:43:36.939+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-30T17:43:36.940+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --profiles-dir /opt/***/DBT_Project  --project-dir /opt/***/dbt_trip --models green_trip']
[2024-09-30T17:43:36.956+0000] {subprocess.py:86} INFO - Output:
[2024-09-30T17:43:39.756+0000] {subprocess.py:93} INFO - [0m17:43:39  Running with dbt=1.8.0
[2024-09-30T17:43:40.082+0000] {subprocess.py:93} INFO - [0m17:43:40  Registered adapter: postgres=1.8.0
[2024-09-30T17:43:40.454+0000] {subprocess.py:93} INFO - [0m17:43:40  Unable to do partial parsing because of a version mismatch
[2024-09-30T17:43:42.810+0000] {subprocess.py:93} INFO - [0m17:43:42  [[33mWARNING[0m]: Deprecated functionality
[2024-09-30T17:43:42.810+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-09-30T17:43:42.810+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-09-30T17:43:42.811+0000] {subprocess.py:93} INFO - information.
[2024-09-30T17:43:43.122+0000] {subprocess.py:93} INFO - [0m17:43:43  Encountered an error:
[2024-09-30T17:43:43.123+0000] {subprocess.py:93} INFO - Compilation Error
[2024-09-30T17:43:43.123+0000] {subprocess.py:93} INFO -   Model 'model.fact_trip.zone_lookup' (models/Staging/zone_lookup.sql) depends on a source named 'staging.stg_zone_lookup' which was not found
[2024-09-30T17:43:44.122+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-09-30T17:43:44.123+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-30T17:43:44.135+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-09-30T17:43:44.140+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Taxi_trip_pipeline, task_id=dbt_staging_layer.dbt_run_green_trip, run_id=manual__2024-09-30T17:37:19.589455+00:00, execution_date=20240930T173719, start_date=20240930T174336, end_date=20240930T174344
[2024-09-30T17:43:44.154+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 48 for task dbt_staging_layer.dbt_run_green_trip (Bash command failed. The command returned a non-zero exit code 2.; 266)
[2024-09-30T17:43:44.179+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-09-30T17:43:44.200+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-30T17:43:44.202+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
